content/en/community/_index.md

```diff
@@ -26,5 +26,6 @@ Here are some articles written by and for members of the Navidrome community. If
 * [Servidor de música. La alternativa a Airsonic es Navidrome](https://ugeek.github.io/post/2020-09-17-servidor-de-musica-la-alternativa-a-airsonic-es-navidrome-.html?utm_source=feedburner&utm_medium=feed&utm_campaign=Feed%3A+ugeek+%28uGeek%29&utm_content=FeedBurner) - Spanish podcast, 16/Sep/2020 by uGeek
 * [スマホでの音楽環境のためメディアサーバを探した話 (A story about searching for a media server)](https://blog.asterism.xyz/posts/2020-09-02/) - Japanese blog post, 02/Sep/2020 by aries
 * [Les Tutos: Docker No. 14: Navidrome - Serveur de streaming de musique](https://www.youtube.com/watch?v=FJ0pFRPdVjs) - French installation video, 25/Apr/2020 by Tonton Jo
+* [Navidrome：自建音乐库的又一选择](https://www.himiku.com/archives/navidrome.html) - Chinese (simplified) blog post, 25/Nov/2023 by mikusa
```

Comment:
Please move your entry to the top of this list. The list is sorted by date, descending.